Sources/Purchasing/StoreKitAbstractions/StoreProduct.swift

```diff
@@ -94,8 +94,10 @@ public typealias SK2Product = StoreKit.Product
     // switflint:enable missing_docs
 }
 
+@_spi(Internal) extension StoreProduct: StoreProductType { }
+
 /// Type that provides access to all of `StoreKit`'s product type's properties.
-internal protocol StoreProductType: Sendable {
+@_spi(Internal) public protocol StoreProductType: Sendable {
```

Comment:
Same question about why this now needs to be exposed, and whether we can get around just using TestStoreProduct. (Probably not but curious of the reason)